rio_tiler/profiles.py

```diff
@@ -52,12 +52,12 @@ def __init__(self):
     def get(self, key, default=None):
         """Like normal item access but return a copy of the key."""
         if key in (self.keys()):
-            return self.data[key].copy()
+            return deepcopy(self.data[key])
```

Comment:
gives the same results (for tests) but I think it's cleaner/safer

Reply:
It depends if `self.data[key]` ever has deeply-nested objects.

Reply:
I don't think so 🤷‍♂️ I guess we can revert